src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,24 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+
+    private List<Car> cars;
+    public Cars() {
```

Comment:
멤버 변수와 생성자 사이는 구분을 위해 개행을 추가하는 것이 좋습니다.
이래야 깔끔해집니다.
```suggestion
    private List<Car> cars;

    public Cars() {
```

Reply:
일급컬렉션으로 생성한 이유를 들어볼 수 있을까요?
Cars라는 클래스를 봤을 때 `List<Car>`를 사용했을 때와 차이점이 없어보여요

Reply:
먼저 디테일한 피드백 감사합니다.  말씀하신대로 일급컬렉션에 대해서 큰 차이가 없는 상태입니다. 그럼에도 불구하고 일급 컬렉션을 사용한 이유는 일급 컬렉션을 언제 사용해야 하는지, 어떻게 사용하는지에 대한 기준점이 아직 모호해서 이번에 일급 컬렉션으로 프로그래밍 해보자 하여 사용 해봤습니다. 그런데 역시나 아직 언제 사용해야 하는지는 직관적으로 잘 모르겠습니다ㅠ